clients/js/src/aptos.ts

```diff
@@ -325,3 +328,71 @@ function hex(x: string): Buffer {
     "hex"
   );
 }
+
+export async function query_registrations_aptos(
+  network: "MAINNET" | "TESTNET" | "DEVNET",
+  module: "Core" | "NFTBridge" | "TokenBridge"
+): Promise<Object> {
+  const n = NETWORKS[network]["aptos"];
+  const client = new AptosClient(n.rpc);
+  const contracts = CONTRACTS[network]["aptos"];
+  let state_object_id: string;
+
+  switch (module) {
+    case "TokenBridge":
+      state_object_id = contracts.token_bridge;
+      if (state_object_id === undefined) {
+        throw Error(`Unknown token bridge contract on ${network} for Aptos`);
+      }
+      break;
+    default:
+      throw new Error(`Invalid module: ${module}`);
+  }
+
+  console.log(
```

Comment:
I know this is a draft still, but I think it would be nice for this command to output valid json (without any additional logging)

Reply:
I addressed this comment in #3035 which replaces this PR.